.github/workflows/integration_tests.yml

```diff
@@ -69,6 +69,9 @@ jobs:
           - build_type: centaurHoricromtalPapiV2beta
             build_mysql: 8.4
             friendly_name: Centaur Horicromtal PapiV2 Beta with MySQL 8.4
```

Comment:
Maybe we can remove the old one straight away; since Horicromtal primarily tests distribution of workflows between runners, not the backend itself

Reply:
I am little hesitant to increase the scope of this PR. i think we can do the removal of Papi related centaur tests together in a separate PR later.

Reply:
Seems reasonable to me.